src/sentry/api/endpoints/organization_member/index.py

```diff
@@ -402,23 +418,32 @@ def post(self, request: Request, organization) -> Response:
             )
             save_team_assignments(om, teams)
 
-        if settings.SENTRY_ENABLE_INVITES and result.get("sendInvite"):
-            referrer = request.query_params.get("referrer")
-            om.send_invite_email(referrer)
-            member_invited.send_robust(
-                member=om, user=request.user, sender=self, referrer=request.data.get("referrer")
+        if is_active_staff(request):
+            self.create_audit_entry(
```

Comment:
From the ticket I thought we're completely skipping the audit log for staff. It's worth double checking

Reply:
In the slack conversation we had, I believe @leedongwei  mentioned we should still emit the log.